app/src/main/java/ch/epfl/sweng/erpa/model/User.java

```diff
@@ -0,0 +1,31 @@
+package ch.epfl.sweng.erpa.model;
+
+public class User {
```

Comment:
This class looks more like an `UserAuth` to me, am I wrong? Else, please rename it.
It's a good practice to add a comment on top of models describing precisely what they represent.